armi/reactor/reactors.py

```diff
@@ -348,6 +379,18 @@ def symmetry(self) -> geometry.SymmetryType:
         .. impl:: Get core symmetry.
             :id: I_ARMI_R_SYMM
             :implements: R_ARMI_R_SYMM
+
+            This property getter returns the symmetry attribute of the
+            spatialGrid instance attribute.  The spatialGrid is an instance of a
+            child of the abstract base class :py:class:`Grid
+            <armi.reactor.grids.grid.Grid>` type.  The symmetry attribute is an
+            instance of the :py:class:`SymmetryType
+            <armi.reactor.geometry.SymmetryType>` class, which is a wrapper
+            around the :py:class:`DomainType <armi.reactor.geometry.DomainType>`
+            and :py:class:`BoundaryType <armi.reactor.geometry.BoundaryType>`
+            enumerations used to classify the domain (e.g., 1/3 core, quarter
+            core, full core) and symmetry boundary conditions (e.g., periodic,
```

Comment:
1/3 core works only with periodic boundary condition. Quarter core works with either periodic or reflective boundary condition.  We may want to make it clear here.